src/linear_samplers/block_col_rand_replace.jl

```diff
@@ -0,0 +1,46 @@
+"""
+    LinSysVecBlkColReplace <: LinSysBlkColSampler 
+
+A mutable structure with fields to handle randomly subset block sampling with replacement, also known as
+vanilla block randomized kaczmarz. 
+
+# Fields
+- `block_size::Int64` - Specifies the size of each block.
+- `block::Vector{Int64}` - The list of all the rows in each block.
```

Comment:
It was supposed to be a field for memory purposes.  It may not be strictly necessary at this current moment, but it does not hurt anything and if we switch to using `StatsBase.sample` as we had discussed then we will need it to be part of the structure.

Reply:
I also changed the description of the first comment. Specifically I say "A mutable structure with fields to store information for a sampling method that  forms a new block by uniformly sampling rows of `A` without replacement"